cpp/src/io/orc/writer_impl.cu

```diff
@@ -379,11 +375,11 @@ __global__ void copy_string_data(char* string_pool,
 }  // namespace
 
 void persisted_statistics::persist(int num_table_rows,
-                                   bool single_write_mode,
+                                   SingleWriteMode single_write_mode,
```

Comment:
According to https://github.com/rapidsai/cudf/blob/branch-23.06/cpp/doxygen/developer_guide/DEVELOPER_GUIDE.md#code-and-documentation-style-and-formatting, We use snake_case to name enums.